bin/_mocha

```diff
@@ -642,11 +642,19 @@ if (program.watch) {
   runner = mocha.run(program.exit ? exit : exitLater);
 }
 
-process.on('SIGINT', () => {
-  runner.abort();
-
-  // This is a hack:
-  // Instead of `process.exit(130)`, set runner.failures to 130 (exit code for SIGINT)
-  // The amount of failures will be emitted as error code later
-  runner.failures = 130;
+process.on('SIGINT', function forceStop() {
+  if (forceStop.notCalled === undefined) {
+    runner.abort();
+
+    // This is a hack:
+    // Instead of `process.exit(130)`, set runner.failures to 130 (exit code for SIGINT)
+    // The amount of failures will be emitted as error code later
+    runner.failures = 130;
+    forceStop.notCalled = false;
+    // invoke SIGINT after 2 seconds
```

Comment:
So why wait? Mocha processing is done -- timeout `0`.

Reply:
Thank you!!
Seems there is a lot of change happening in codebase..
Have modified based on the comments..